packages/server/src/whip/sfu/sfuWhipResource.ts

```diff
@@ -383,22 +392,6 @@ export class SfuWhipResource implements WhipResource {
     return Promise.resolve(this.answer);
   }
 
-  assignBroadcasterClients(broadcasterClientSfuPairs: BroadcasterClientSfuPair[]) {
```

Comment:
Can we keep these two functions but mark as deprecated?
There are already some examples and components out there that uses them.